### DISCUSSION TITLE
How to refetch when window does not have focus

### DISCUSSION THREAD

**Discussion by henrikq** [TanStack/query#2867] [Q&A]
I want refetch a query, even if the window does not have focus. As far as i can tell `queryClient.refetchQueries` does not do this. What is the best/simplest way to accomplish this?

**Comment by TkDodo** [comment]
I currently don't see any indication that `refetchQueries` would only work for focused windows. Do you have a codesandbox reproduction please?

You could of course tinker with the [FocusManager](https://react-query.tanstack.com/reference/focusManager), but I don't think it should be needed.

The only thing we are currently doing is:
-  pause failed queries if the window is not active
- refetch if it gets the focus if refetchOnWindowFocus is turned on
- use it to stop refetchInterval if the window is not active (unless explicitly opted in via `refetchIntervalInBackground`.
